### PR TITLE
feat: redux added

### PR DESCRIPTION
**Pull request by dev-ashishsethi**
refactored my code with redux toolkit

files to check:
store/index.js
index.js

### CODE REVIEW

**Review by m26unkwn — APPROVED** [review]
Code is good and Nice that you have shifted your global state to redux state management

File: src/Pages/WatchLater/WatchLater.jsx [hunk 1]
```diff
@@ -1,29 +1,35 @@
...
-    watchLaterPage,
-    setHistoryPage,
-    setWatchLaterPage,
-    setPlaylistPage,
+    // watchLater,
+    // setWatchLaterVideos,
+    // watchLaterPage,
+    // setHistoryPage,
+    // setWatchLaterPage,
+    // setPlaylistPage,
```
Comment (saurabhchirde) [inline src/Pages/WatchLater/WatchLater.jsx:14]: remove comments

File: src/Utils/Playlist/addVideoToPlaylist.js [hunk 2]
```diff
@@ -1,31 +1,49 @@
...
+  // setPlaylists((playlists) =>
+  //   playlists.map((playlist) => {
+  //     if (playlist._id === res.response.playlist._id) {
+  //       playlist.videos = res.response.playlist.videos;
+  //       return playlist;
+  //     } else {
+  //       return playlist;
+  //     }
+  //   })
+  // );
```
Comment (saurabhchirde) [inline src/Utils/Playlist/addVideoToPlaylist.js:47]: remove comments

File: src/Utils/Playlist/deleteVideoFromPlaylist.js [hunk 3]
```diff
@@ -1,26 +1,41 @@
...
+    // setPlaylists((playlists) =>
+    //   playlists.map((playlist) => {
+    //     if (playlist._id === res.response.playlist._id) {
+    //       playlist.videos = res.response.playlist.videos;
+    //       return playlist;
+    //     } else {
+    //       return playlist;
+    //     }
+    //   })
+    // );
```
Comment (saurabhchirde) [inline src/Utils/Playlist/deleteVideoFromPlaylist.js:38]: remove comments

File: src/store/index.js [hunk 4]
```diff
@@ -0,0 +1,110 @@
...
+export const playlistAction = playlistSlice.actions;
+export const watchLaterAction = watchLaterSlice.actions;
+export const historyAction = historySlice.actions;
+
+export const store = configureStore({
+  reducer: {
+    likedVideos: likedVideosSlice.reducer,
+    playlist: playlistSlice.reducer,
+    watchLater: watchLaterSlice.reducer,
+    history: historySlice.reducer,
```
Comment (saurabhchirde) [inline src/store/index.js:108]: you can create separate files for all the slices and import them into the store, instead of writing them in the same file.

**Review by saurabhchirde — APPROVED** [review]
Good to see that you have added Redux to your project. Gave some suggestions for better code practice. After resolving them, you can proceed to merge.

**Pull request merged**